src/cpp/benders/factories/BendersFactory.cpp

```diff
@@ -49,18 +52,16 @@ int RunBenders(char** argv, const std::filesystem::path& options_file,
       writer = build_void_writer();
     }
 
+    world.barrier();
     pBendersBase benders;
     if (method == BENDERSMETHOD::BENDERS) {
       benders = std::make_shared<BendersMpi>(benders_options, logger, writer,
                                              env, world);
-    } else if (method == BENDERSMETHOD::BENDERSBYBATCH) {
+    } else {
```

Comment:
Since BENDERSMETHOD is an enum (good), it would be best to employ a switch case and keep the "else" clause in a default case.
It's "just in case" a new value need implementation

Reply:
By the way, the enum also has a possible value `MERGEMPS` (but I think that if it is the case, we should never end up here, but rather call the merge_mps executable)

Reply:
two things:
the "method" option in CLI (xpansion-launcher) expecte either (benders or mergemps) while the fullrun or the benders factory set only BENDERSMETHOD to {BENDERS or *BYBATCH}
i will clarify usage

Reply:
used switch case  && deleted MERGE_MPS and